test/extended/builds/gitauth.go

```diff
@@ -91,7 +93,9 @@ var _ = g.Describe("[sig-builds][Feature:Builds][Slow] can use private repositor
 
 				ctx := context.Background()
 				sshKey, err := oc.AsAdmin().KubeClient().CoreV1().Secrets("build-e2e-github-secrets").Get(ctx, "github-ssh-privatekey", metav1.GetOptions{})
-				o.Expect(err).NotTo(o.HaveOccurred())
+				if err != nil {
+					g.Skip("test requires secret only available in CI step registry")
+				}
```

Comment:
Check for 404/not found:

```suggestion
				if errors.IsNotFound(err) {
					g.Skip("required secret build-e2e-github-secrets/github-ssh-privatekey is missing")
				}
```